miqi/bridge/loop.py

```
@@ -736,6 +736,22 @@ async def _chat_send_handler(
+        # metadata so the KUN loop reads it on every model step (max_tokens,
+        # round fuse, search fan-out all key off thread.metadata.mode).
+        # Frontend sends reasoning_mode (preload 7th param); legacy "mode"
+        # carries executionPolicy (plan/manual/edit/auto) — never both.
+        mode_param = params.get("reasoning_mode") or params.get("mode")
+        if mode_param in ("fast", "think") and runtime is not None:
+            try:
+                thread = await runtime.thread_store.get(thread_id)
+                if thread is not None:
+                    thread.setdefault("metadata", {})["mode"] = mode_param
+                    await runtime.thread_store.upsert(thread)
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)